google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleStage.java

```diff
@@ -62,23 +68,29 @@ public AppEngineFlexibleStage(
    */
   public void stage(@NotNull Path stagingDirectory) {
     try {
+      AppEngineFlexibleFacet flexibleFacet =
+          AppEngineFlexibleFacet.getFacetByModuleName(
+              deploymentConfiguration.getModuleName(), project);
+
+      if (flexibleFacet == null
+          || !Files.exists(Paths.get(flexibleFacet.getConfiguration().getAppYamlPath()))) {
+        throw new RuntimeException(
```

Comment:
here too it would be nice to differentiate between the error conditions and have more helpful messages.